.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Use this to provide a version, instead of deriving it from the changelog.
+        required: false
+        type: string
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      # The current realm studio release process keeps changes for the upcoming release
+      # in a separate file (RELEASENOTES.md), and copies its content on top of the
+      # changelog, before resetting it from a template.
+      # The action used by other components keeps the release notes directly in
+      # CHANGELOG.md, and just fills in the version. The format is slightly different
+      # - but should be close enough that switching should be possible
+      - name: Update Changelog
+        id: update-changelog
+        uses: realm/ci-actions/update-changelog@fa20eb972b9f018654fdb4e2c7afb52b0532f907
+        with:
+          changelog: ${{ github.workspace }}/CHANGELOG.md
+          version: ${{ inputs.version }}
+      - name: Update package.json and package-lock.json
+        run: |
+          sed -i 's/"version": .*/"version": "${{ steps.update-changelog.outputs.new-version }}",/' package.json
+          npm ci --legacy-peer-deps
+          npm i --package-lock-only
```

Comment:
Why is this needed after the "npm ci"?

Reply:
I had problems with package-lock.json not getting updated in my runs - not entirely sure why.

Reply:
I'd appreciate we found the root cause of that.

Reply:
Is it related to https://github.com/npm/cli/issues/5113?